server/src/services/user.ts

```diff
@@ -0,0 +1,34 @@
+import { prisma } from "../utils/prisma.db";
+
+const get = async (id: string) => {
+  const user = await prisma.user.findUniqueOrThrow({
+    where: { id: Number(id) },
```

Comment:
Fun note: a middleware can be added in a later PR that does this so it does not have to be done in all of the service methods.

Reply:
You can resolve this convo yourself 😄